content/automate/ManageIQ/Service/Generic/StateMachines/GenericLifecycle.class/__methods__/update_status.rb

```diff
@@ -46,24 +46,34 @@ def service_action
                 end
               end
 
-              def update_status(service)
-                # Get status from input field status
-                status = @handle.inputs['status']
+              def error
+                @handle.root['ae_result'] == "error"
+              end
 
+              def updated_message(status)
                 updated_message = "Server [#{@handle.root['miq_server'].name}] "
                 updated_message += "Service [#{service.name}] #{service_action} "
```

Comment:
@tinaafitz the service method is being called here to fetch the service object, if someone wants to log another property from the service object, they would call service.some_property that would end up calling the method again. So can we pass the service into this method